.buildkite/pipeline.yml

```diff
@@ -1,8 +1,13 @@
 # Nodes with values to reuse in the pipeline.
 common_params:
   # Common plugin settings to use with the `plugins` key.
-  - &common_plugins
-    - automattic/a8c-ci-toolkit#2.17.0
+  - &ci_toolkit
+    automattic/a8c-ci-toolkit#2.17.0
```

Comment:
PS: Totally unrelated to this PR, but FYI `2.18.2` is the latest version nowadays, in case you want to use the occasion to update the plugin while at it (or in a future PR) 🙃